dashboard.php

```diff
@@ -67,51 +108,18 @@
         <h2 class="section-title">Completed Tests</h2>
         <!-- Area of page that contains the test boxes -->
         <div class="test-section">
+        <?php foreach ($completedTests as $test): ?>
```

Comment:
Add a check similar to the Tests To Complete to show a message saying they have no completed tests to show if the `completedTests` is empty.